src/uint/boxed/inv_mod.rs

```diff
@@ -13,14 +13,78 @@ impl BoxedUint {
     }
 
     /// Computes 1/`self` mod `2^k`.
+    /// This method is variable w.r.t. `self` and `k`.
     ///
     /// If the inverse does not exist (`k > 0` and `self` is even),
     /// returns `Choice::FALSE` as the second element of the tuple,
     /// otherwise returns `Choice::TRUE`.
-    pub(crate) fn inv_mod2k(&self, k: u32) -> (Self, Choice) {
+    pub(crate) fn inv_mod2k_full_vartime(&self, k: u32) -> (Self, Choice) {
```

Comment:
Yeah, there are various permutations of what "vartime" means and I've not been sure how to handle when we want more than one. This seems fine, especially for `pub(crate)`